src/components/TransactionItemRow/DataCells/TypeCell.tsx

```diff
@@ -23,18 +25,19 @@ const getTypeIcon = (type?: string) => {
     }
 };
 
-const getTypeText = (type?: string) => {
+const getTypeText = (type?: string): TranslationPaths => {
     switch (type) {
         case CONST.EXPENSE.TYPE.CASH_CARD_NAME:
-            return 'Cash';
+            return 'iou.cash';
         case undefined:
-            return 'Cash';
+            return 'iou.cash';
```

Comment:
NAB do we need this specific case? Can't we just use the default?

Reply:
will see if default handles it, but most likely we can make improve it